src/templateselector.cpp

```diff
@@ -614,3 +625,15 @@ void TemplateSelector::openTemplateLocation()
 	}
 }
 
+void TemplateSelector::onRequestError()
+{
+    QNetworkReply *reply = qobject_cast<QNetworkReply *>(sender());
+    if (!reply) return;
+
+    QMessageBox::warning(this, tr("Browse template repository"),

```

Comment:
blocking message boxes again
I would rather prefer to have a message in the tree view.

Reply:
Thanks for pointing out this (again). I resolve this, see below